docs/src/getting_started.ipynb

```diff
@@ -51,13 +61,66 @@
     "import numpy as np"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "5e9bf8e3-07fd-4e4d-bdb8-d32a48a366c7",
+   "metadata": {},
+   "source": [
+    "## Part 1: Events and EventSets\n",
+    "\n",
+    "The most basic unit of data in Temporian is an **event**. An event consists of a timestamp and a set of feature values.\n",
```

Comment:
**Events** are the basic unit of data in Temporian. They consist of a timestamp and a set of feature values. Events are not handled individually, but are instead grouped together into **EventSets**.